pom.xml

```diff
@@ -881,6 +881,7 @@
                         <javaVersion>1.8</javaVersion>
                         <failOnViolations>true</failOnViolations>
                         <exclusionsFile>${air.main.basedir}/src/modernizer/exclusions.txt</exclusionsFile>
+                        <violationsFile>${air.main.basedir}/src/modernizer/violations.xml</violationsFile>
```

Comment:
Add comment above `<version>1.4.0</version>` that `violationsFile` that violations file needs to be updated to match new version.
If we cannot do better.